### ISSUE TITLE
Attribute classification

### ISSUE THREAD

**Issue by ghost** [LynnHo/AttGAN-Tensorflow#41]
Hi, i need to label my new set of custom images attributes to use AttnGAN.

Is there any pretrained model for attribute classification?

celebA [poster](https://liuziwei7.github.io/papers/faceattributes_poster.pdf) suggests below website..

http://personal.ie.cuhk.edu.hk/~lz013/projects/FaceAttributes.html

above link isn't working.

could you share resources or pretrained model to classify attributes if you've any..

**Comment by LynnHo** [comment]
@VijayRameshkumar You can find a classifier here https://github.com/LynnHo/AttGAN-Tensorflow/issues/11#issuecomment-490842498.
